README.md

```
@@ -1 +1,346 @@
+<br/>
+
+ </details>
+
+
+## 📢 코딩 (개발) 컨벤션
+
+ <details> 
+	 
+### 변수
+
+- var 금지.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)